include/crm_internal.h

```diff
@@ -100,6 +100,7 @@
 #define PCMK__XA_SCHEMAS                "schemas"
 #define PCMK__XA_SRC                    "src"
 #define PCMK__XA_SUBT                   "subt"                  // subtype
+#define PCMK__XA_T                      "t"                     // type
```

Comment:
Maybe as a wishlist item we can change some of the most obfuscated internal literals and maintain compatibility for a while by checking old and new.

Definitely not worth the detour right now, and it is a pain to maintain, but might be worthwhile for the worst ones. I'd love to see `PCMK_XA_TYPE` instead of `PCMK_XA_T` in the future, for example.

Reply:
Yeah it would be nice but ugly for a long transition period. Similarly I'd like to avoid using `__function__` as an XML element name so we can (eventually) check for known names rather than just assume children are as expected.